packages/edit-navigation/README.md

```diff
@@ -59,13 +61,26 @@ This unlocks significant additional capabilities in the Navigation Editor. For e
 
 #### Technical Implementation details
 
-By default, `core/navigation-link` items are serialized and persisted as `nav_menu_item` posts. No serialized block HTML is stored for these standard link blocks.
+There are two types of item that can be added to a navigation:
 
-_Non_-link navigation items however, are [persisted as `nav_menu_items` with a special `type` of `block`](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/packages/edit-navigation/src/store/utils.js#L159-L166). These items have an [_additional_ `content` field which is used to store the serialized block markup](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/lib/navigation.php#L71-L101).
+1. Navigation links - these utilise the `core/navigation-link` block and are _not_ persisted as blocks.
+2. Block-based links - these are any item that is _not_ a `core/navigation-link` block. They are persisted as blocks (via a wrapper - see below).
+
+By default, Navigation link items are serialized and persisted as `nav_menu_item` posts. No serialized block HTML is stored for these standard link blocks.
+
+Block-based link items however, are [persisted as `nav_menu_items` with a special `type` of `block`](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/packages/edit-navigation/src/store/utils.js#L159-L166). These items have an [_additional_ `content` field which is used to store the serialized block markup](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/lib/navigation.php#L71-L101).
```

Comment:
It's cool that we want to document how this works today, but it is very likely to not continue down this path because it does not allow us to save attributes of the wrapping navigation block itself.

Reply:
Yup. Fair enough. I'd like to document correct workings for now as that encourages future contributors to write and update docs in order to correct things 😄

Reply:
Also writing this helps to ensure _I_ fully understand things. If you can't explain it you don't understand it.